core/src/Kokkos_Core.hpp

```diff
@@ -122,6 +122,10 @@ void initialize(int& narg, char* arg[]);
 
 void initialize(InitArguments args = InitArguments());
 
+void pre_initialize(InitArguments args = InitArguments());
+
+void post_initialize(InitArguments args = InitArguments());
```

Comment:
Belongs to `Impl::` (or `Experimental::`) namespace.

Reply:
Impl

Reply:
Also no default argument.

Reply:
Fixed.